base/sort.jl

```diff
@@ -219,6 +219,72 @@ for s in [:searchsortedfirst, :searchsortedlast, :searchsorted]
     end
 end
 
+@doc """
```

Comment:
I think so - I followed the example [here](https://docs.julialang.org/en/latest/manual/documentation/#Advanced-Usage-1).